packages/protect/src/config/index.ts

```diff
@@ -0,0 +1,78 @@
+import fs from 'node:fs'
+import path from 'node:path'
+
+/**
+ * A lightweight function that parses a TOML-like string
+ * and returns the `workspace_id` value found under `[auth]`.
+ *
+ * @param tomlString The contents of the TOML file as a string.
+ * @returns The workspace_id if found, otherwise undefined.
+ */
+function getWorkspaceId(tomlString: string): string | undefined {
```

Comment:
Why is this function needed? It should all be handled by protectjs-ffi

Reply:
Need it to get the workspace ID when getting a CTS token through the main Protect.js interface